test/index_test.js

```diff
@@ -78,6 +78,11 @@ describe("Kinto", () => {
       expect(new Kinto({requestMode}).collection("x").api.http.requestMode)
         .eql(requestMode);
     });
+
+    it("should propagate the prefixDB option to child dependencies", () => {
+      expect(new Kinto({prefixDB: "app--"}).collection("x").db.dbname)
+        .eql("app--default/x");
```

Comment:
Makes me think we should probably pay attention to possible db/store naming constraints in different adapaters (IDB/localStorage)… Maybe we should provide some validation for these?